tsconfig.json

```diff
@@ -12,6 +12,7 @@
     /* Additional Checks */
     "noUnusedLocals": true,
     "noUnusedParameters": true,
+    "noImplicitOverride": true,
```

Comment:
Type tests on TS 4.2 do not work with this option. I could move it to `tsconfig.json` inside package directories. Not sure if that is worth it.